src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -768,6 +768,14 @@ To add a post-script to a task, use:
 #%post_script(name=TASK_NAME, language=SCRIPT_LANGUAGE, [path=./POST_SCRIPT_FILE.py])
 ```
 
+==== Deleting a task
+
+To delete a task from the workflow, the user should run the pragma `#%delete_task()` in the following way:
```

Comment:
To delete a task from a workflow ?

Reply:
I used 'the' because we can create one and only one workflow in a notebook (but submit multiple jobs).